pages/common/mcat.md

```diff
@@ -0,0 +1,32 @@
+# mcat
+
+> Parse, convert, and preview files (including Markdown), directories, images, and videos.
+> More information: <https://github.com/Skardyy/mcat>.
+
+- Display the contents of a file:
+
+`mcat {{path/to/file}}`
+
+- Display a Markdown file with a specific theme:
+
+`mcat --theme {{theme_name}} {{path/to/file.md}}`
```

Comment:
```suggestion
`mcat --theme {{theme_name}} {{path/to/file}}.md`
```

Reply:
The page [glow.md](https://github.com/tldr-pages/tldr/blob/main/pages/common/glow.md) uses the `{{path/to/file.md}}` syntax, so I updated it to follow the same format.